pomegranate/NaiveBayes.pyx

```diff
@@ -93,8 +95,9 @@ cdef class NaiveBayes(BayesModel):
 
 	@classmethod
 	def from_samples(cls, distributions, X, y=None, weights=None,
-		pseudocount=0.0, stop_threshold=0.1, max_iterations=1e8,
-		callbacks=[], return_history=False, verbose=False, n_jobs=1):
+		pseudocount=0.0, alpha: Optional[float] = None, stop_threshold=0.1,
```

Comment:
Would you mind removing the typing? Regardless of whether typing is good or bad (it's own debate), partially typing pomegranate seems like a bad idea and I can't commit to typing every parameter.